### DISCUSSION TITLE
OPENFAST parameter modification

### DISCUSSION THREAD

**Discussion by sjihf12** [OpenFAST/openfast#1706] [Q&A]
Dear Professor
Does openfast provide the MATLAB interface file for modifying file parameters？ For example, I want to modify some parameters in the file "NRELOffshrBsline5MW_OC3Hywind_MoorDyn" through the MATLAB interface file.

**Comment by jjonkman** [comment]
Dear @sjihf12,

I'm not sure what you mean when you refer to a "MATLAB interface file"; please clarify.

Best regards,

**Reply by sjihf12** [reply]
I'm sorry that I didn't describe it clearly. I want to use MATLAB program to modify offshore wind turbine parameters in the file. Does openfast provide MATLAB program to modify offshore wind turbine parameters in the file？

**Reply by jjonkman** [reply]
Dear @sjihf12,

Scripts available in the MATLAB Toolbox (https://github.com/OpenFAST/matlab-toolbox) or Python toolbox (https://github.com/OpenFAST/python-toolbox) can be used to read and write FAST input files.  See related conversations here: https://github.com/OpenFAST/openfast/issues/658 and https://github.com/OpenFAST/openfast/issues/573.

Best regards,

**Reply by sjihf12** [reply]
Thank you！
